server/odc-service/src/main/java/com/oceanbase/odc/service/dlm/TaskGeneratorMapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.dlm;
+
+import org.apache.commons.lang.StringUtils;
+
+import com.oceanbase.odc.metadb.dlm.TaskGeneratorEntity;
+import com.oceanbase.tools.migrator.common.dto.TaskGenerator;
+import com.oceanbase.tools.migrator.common.element.PrimaryKey;
+import com.oceanbase.tools.migrator.core.handler.genarator.GeneratorStatus;
+import com.oceanbase.tools.migrator.core.handler.genarator.GeneratorType;
+
+/**
+ * @Author：tinker
+ * @Date: 2023/8/10 15:47
+ * @Descripition:
+ */
+public class TaskGeneratorMapper {
+
+
+    public static TaskGenerator entityToModel(TaskGeneratorEntity entity) {
```

Comment:
there is no need to do this, you may use `@Mapper`, just like what `ConnectionMapper` did